Driverless AI/disparate-impact-analysis-tutorial/disparate-impact-analysis-tutorial.md

```diff
@@ -0,0 +1,458 @@
+
+4. Scroll down and let's take a look at *Group Disparity*.
+
+![0.8-low-threshold](assets/0.8-low-threshold.jpg)
+
+You will notice that your benchmark (1/Male) should be one as it provides the level to compare. In financial services, there is an ‘accepted’ rule of thumb that, as a benchmark, one class should not be treated 80% less favorably than another, as a starting point. This benchmark is up for debate, and therefore, we will see how this will develop going forward, but for now, we have set the low (unfairness) threshold to *.8** (it should already be set to .8). As a result, with a .8 low threshold, we can see that none of the group disparity metrics are less than point 8, which is a good sign.
+
+5. Now let’s adjust the *low threshold* to .9 and see what happens: 
+
+![0.9-low-threshold](assets/0.9-low-threshold.jpg)
+
+We can see that if we adjust the low threshold, the group disparity metrics will become highlighted and begin to flag, saying that if .9 is the cutoff,  this class (2/Women) will be treated unfairly by the model. Note: the *True Positive Rate Disparity* and *False Negative Rate Disparity* are not affected.
+
+6. Let’s scroll down and investigate *Group Parity* and check if all classes are being treated fairly by the model under the benchmark thresholds.
+
+![group-parity](assets/group-parity.jpg)
+
+Here we can see *True* across all classes and metrics, which is what we want to see.
+
+### Deeper Dive and Resources
+
+- [9] [WHAT IS ADVERSE IMPACT? AND WHY MEASURING IT MATTERS](https://www.hirevue.com/blog/what-is-adverse-impact-and-why-measuring-it-matters)
+
+- [10] [Encyclopedia of Systems Biology](https://link.springer.com/referenceworkentry/10.1007%2F978-1-4419-9863-7_255)
+
+## Task 5: Sensitivity Analysis Part 1
+
+1. Let’s start up a new experiment with the same dataset as before. Keep the settings & target variable the same; however, this time, let’s keep all the columns in the dataset. 
+
+![new-experiment](assets/new-experiment.jpg)
+
+After the experiment is over:
+
+ - Click on *INTERPRET THIS MODEL*
+
+ - After, in the *DAI Models* tab you should click on the Sensitivity Analysis option 
+
+After that, you should land on our Sensitivity Analysis Dashboard: 
+
+![SA-UI](assets/SA-UI.jpg)
+
+Some things to notice:
+
+  1. In our *Summary* information for the dataset located on the left side of the dashboard, we, in particular, can see our chosen cutoff metric, and the number for that metric. 
+
+  ![SA-UI-Summary](assets/SA-UI-Summary.jpg)
+
+  * In our *Summary* information for the dataset located on the left side of the dashboard, we, in particular, can see our chosen cutoff metric, and the number for that metric. Our * CUTOFF* is 0.2676... Anything below the * CUTOFF* will mean the model predicts a customer will not default, while anyone greater than or equal to the * CUTOFF* will default. 
```

Comment:
Remove the space in between * CUTOFF* so that it can be bolded. I believe there is three * CUTOFF* in this paragraph.

Reply:
@Castro38 done